src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -349,9 +350,12 @@ impl<'a, 'b> BuildReducedGraphVisitor<'a, 'b> {
 
         self.r.indeterminate_imports.push(directive);
         match directive.subclass {
+            // Don't add unresolved underscore imports to modules
+            SingleImport { target: Ident { name: kw::Underscore, .. }, .. } => {}
```

Comment:
What happens if this special case is removed? Some diagnostics regress?

Reply:
I don't think so, we just won't be able to remove it in `resolve_imports.rs:831`.

Reply:
Ah, I see, the disambiguator is not available in that context.